src/l10n.py

```diff
@@ -119,8 +128,10 @@ def getTranslationLanguage():
         return language
 
     return userlocale
-    
+
+
 def getWindowsLocale(posixLocale):
+    """Get the Windows locale"""
```

Comment:
Technically this converts the locale string from UNIX to Windows format, because they use different ones in their libraries. E.g. "en_EN.UTF-8" to "english".